lib/task/sfBaseTask.class.php

```diff
@@ -17,6 +17,13 @@
  */
 abstract class sfBaseTask extends sfCommandApplicationTask
 {
+
+    protected $version;
+    protected $namespace;
```

Comment:
namespace, name, briefDescription, detailedDescription fields are defined here:
https://github.com/FriendsOfSymfony1/symfony1/blob/6354fd37141c8fd40af563041674b5fe07ca99de/lib/task/sfTask.class.php#L18-L22

Reply:
Oh yes. Sorry, looks like $version is something custom i added long time ago to some tasks. Removed, no deprecated messages anymore.